frontend/src/containers/HomePage.js

```diff
@@ -143,7 +183,7 @@ export class HomePage extends Component {
               </div>
             </div>
           </div>
-          <a href='#'>Learn more about the collectives</a>
+          {false && <a href='#'>Learn more about the collectives</a> }
```

Comment:
Creates unnecessary code to parse through. I can understand if it's significant code or a test that's annoying to rewrite. These seemed like an easy line to delete and bring back when we need it.